class/defaults.yml

```diff
@@ -2,3 +2,12 @@ parameters:
   stackgres_operator:
     =_metadata: {}
     namespace: syn-stackgres-operator
+    charts:
+      stackgres-operator: 1.4.0
+    helmValues:
+      deploy:
+        restapi: false
+    images:
+      kubectl:
+        image: ongres/kubectl
+        tag: v1.24.3-build-6.16
```

Comment:
You don't set a `repository` in the defaults. That means if we deploy this component without  setting a repository you'll patch the image to `/ongres/kubectl:v1.24.3-build-6.16` which is AFAIK not valid.  Either set a default repository as `docker.io` or handle the missing repository in the post processing.